client/src/ui/Form.jsx

```diff
@@ -248,11 +262,14 @@ export default class Form extends Component {
         Session.setPersistent({ "review_num": "" });
         Session.setPersistent({ "courseId": "" });
         this.hide();
-        alert('Thanks for reviewing! New reviews are updated every 24 hours.');
+        //alert('Thanks for reviewing! New reviews are updated every 24 hours.');
```

Comment:
I'm not sure why these comments are still hanging around. They appear in a number of other places as well. It is recommended to get rid of unneeded code.